test/fixtures/config/groups.js

```diff
@@ -0,0 +1,11 @@
+const faker = require('faker');
+
+module.exports = {
+  generate: () => {
+    return {
+      'id' : faker.random.uuid(),
```

Comment:
This won't fail the linter, but keys should not be `'`'d unless it's necessary to do so, and it should not be for any of these cases.